src/app/query/stacks/tokens/token-metadata.utils.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  FtMetadataResponse,
+  NftMetadataResponse,
+  TokenNotFoundResponse,
+} from '@hirosystems/token-metadata-api-client';
+
+export type FtAssetResponse = FtMetadataResponse | TokenNotFoundResponse;
+export type NftAssetResponse = NftMetadataResponse | TokenNotFoundResponse;
+
+function isFtTokenNotFoundResponse(resp: FtAssetResponse): resp is TokenNotFoundResponse {
+  return 'error' in resp;
+}
+
+function isNftTokenNotFoundResponse(resp: NftAssetResponse): resp is TokenNotFoundResponse {
+  return 'error' in resp;
+}
```

Comment:
These two functions are identical, repetition not necessary. 

Suggest `isAssetMetadataNotFound`?

Reply:
I can't ever decide if it is better to repeat bc of needing to us an `or` type?